CONTRIBUTING.md

```diff
@@ -3,15 +3,16 @@
 1.  Request access to this repo
 
 1.  Pull the latest master
-    - On the `master` branch: `git pull`
+
+    * On the `master` branch: `git pull`
```

Comment:
@bhavyaab Can you revert the changes to this file from this PR?

Reply:
I can't because it is done by auto linting on commit. Linting rules need to be fixed.

Reply:
@bhavyaab The linting doesn't actually change any code. It just reads it and checks to make sure things pass. Is your IDE set to make changes?

Reply:
Oh man, I think it's prettier! Sorry about that. It's because I made a change to that doc directly in the browser and prettier never had a chance to run on it, so now it's being run on that file and showing changes.